connect/src/com/telenor/mobileconnect/MobileConnectSdkProfile.java

```diff
@@ -184,17 +133,17 @@ private void tryMccMnc(
                 operatorDiscoveryConfig.getOperatorDiscoveryRedirectUri(),
                 mcc,
                 mnc,
-                odCallbackForMccMnc);
-    }
+                new Callback<OperatorDiscoveryAPI.OperatorDiscoveryResult>() {
```

Comment:
They were actually anonymous all the time (in the sense "anonymous class"), but I needed references in order to reuse them, like passing them as an argument to another method (okay, a could have done with odCallbackForMccMnc only, but I created odCallbackForMsisdn for the sake of clarity) . 

Having scrapped the msisdn-based operator discovery, I do not have to refer explicitely to callbacks any more. Hence the change.